instrumentation/jetty/jetty-8.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jetty/v8_0/JavaLambdaMaker.java

```diff
@@ -9,7 +9,7 @@ public class JavaLambdaMaker {
 
   @SuppressWarnings("FunctionalExpressionCanBeFolded")
   public static Runnable lambda(Runnable runnable) {
-    return runnable::run;
```

Comment:
I think this was written this way to test a runnable that is lambda. We used to not transform lambda classes so they were handled differently from regular runnables. I'll revert this change.